domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewSchedulePersistenceAdapter.kt

```diff
@@ -20,7 +20,23 @@ interface ReviewSchedulePersistenceAdapter {
   fun updateReviewSchedule(updateReviewScheduleDto: UpdateReviewScheduleDto): ReviewSchedule?
 
   /**
-   * Retrieves a [ReviewSchedule] for a given Prisoner. Returns `null` if the [ReviewSchedule] does not exist.
+   * Returns the prisoner's active Review Schedule, where "active" is defined as not having the status "COMPLETED".
+   *
+   * A prisoner will have many Review Schedule records, but only one of them will be their "active" one. It is not
+   * possible for a prisoner to have more than one active Review Schedule.
+   *
+   * A Review Schedule with the status SCHEDULED or one of the EXEMPT_ statuses is considered the active Review Schedule.
+   * A Review Schedule with one of the EXEMPT_ statuses is still consider active as the exemption can be cleared at which
+   * point its status becomes SCHEDULED again.
+   *
+   * Returns `null` if the active [ReviewSchedule] does not exist.
+   */
+  fun getActiveReviewSchedule(prisonNumber: String): ReviewSchedule?
+
+  /**
+   * Retrieves a Prisoner's latest [ReviewSchedule]. The latest (most recently updated) [ReviewSchedule] is returned
+   * irrespective of status.
+   * Returns `null` if no [ReviewSchedule]s exist.
    */
-  fun getReviewSchedule(prisonNumber: String): ReviewSchedule?
+  fun getLatestReviewSchedule(prisonNumber: String): ReviewSchedule?
```

Comment:
We need to be able to get both the active schedule, but also the latest schedule.

The latest schedule might be the last COMPLETED schedule (in the case of the prisoner having had their last review), and we'll need that to be able to return it through the GET endpoint of the REST API.